src/main/resources/org/cbioportal/persistence/mybatisclickhouse/StudyViewMapper.xml

```diff
@@ -83,12 +86,9 @@
             cancer_study_identifier as studyId
         FROM clinical_data_derived
         <where>
-            sample_unique_id IN (
-                <include refid="sampleUniqueIdsFromStudyViewFilter"/>
-                <if test="applyPatientIdFilters == true">
-                    INTERSECT <include refid="getSampleIdsFromPatientIds"/>
-                </if>
-            )
+            <include refid="applyStudyViewFilter">
```

Comment:
we are getting rid of the applyPatientIdFilters condition because for sample clinical data, it's never right to apply a patient filter? hmmm. this always breaks my brain.